test/test_hpyfield.py

```diff
@@ -88,218 +88,3 @@ def EXPORT_PAIR_TYPE(self, *defines):
                 .defines = Pair_defines
             };
```

Comment:
Opps, this test should not have been removed, it's some local debugging/tinkering leftover. Will revert the file back to its master contents.